providers/ldap/ldap_client.go

```diff
@@ -897,7 +898,7 @@ func (l *LClient) userRecord(search *ldap.SearchRequest, lConn *ldap.Conn, name
 		return nilToken, status, fmt.Errorf("User identity not found for Ldap")
 	}
 	token.ExternalAccountID = userIdentity.ExternalId
-	token.AccessToken = userIdentity.ExternalId
+	token.AccessToken = b64.StdEncoding.EncodeToString([]byte(userIdentity.ExternalId))
 	token.OriginalLogin = originalLogin
```

Comment:
@genuss This should get decoded before being used in other methods

Reply:
But how it will be transferred to cattle then? I mean the bug appears when it encodes in header sent to it, so it anyway should be encoded there.

Reply:
@genuss sorry I dont follow your comment, but if you are encoding the value in rancher-auth-service (which then goes encoded to cattle and stored in cattle DB), everytime cattle sends this header to rancher-auth-service back, you need to decode it.

Reply:
@prachidamle  Looks like #57 made it right. I suppose you could merge that PR and reject this one.

Reply:
@genuss ok, thanks.